CHANGELOG.md

```diff
@@ -4,6 +4,17 @@ All notable changes to this project will be documented in this file.
 The format is inspired from [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and the versioning aims to respect [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [2.X.X] - 20XX-XX-XX
+
+### Added
+-CF alternative label/ net capacity factor (#1995)
+
+### Changed
+- apply socio-economic constraints for assessment of potential (#1995)
+
```

Comment:
```suggestion
- net capacity factor (#1995)

```
You changes the class `net capacity factor` by adding an alternative label. You didn't add a new term "CF"... ;)